charts/consul/values.yaml

```diff
@@ -515,6 +515,25 @@ global:
       # @type: string
       secretKey: null
 
+    # job configures settings for the server-acl-init and server-acl-cleanup jobs
+    job:
```

Comment:
This seemed like the least funky place to add this functionality, but I definitely recognize the funkiness and am happy to change it.

Reply:
Could we remove the `job` nesting to make it consistent with other components? e.g. `server.nodeSelector` or `connectInject.nodeSelector`